### DISCUSSION TITLE
Self Hosting on a pi

### DISCUSSION THREAD

**Discussion by GrahamSH-LLK** [plausible/analytics#860] [Self-Hosted Support] [answered]
Hi! I was trying to self-host on a Pi following the instructions in the docs, but I keep getting `standard_init_linux.go:219: exec user process caused: exec format error` errors. How do I fix this?

**Comment by ukutaht** [comment, marked as answer]
Plausible does not work on a Pi. From our docs:

> The server must have a CPU with x86_64 architecture and support for SSE 4.2 instructions.

Raspberry Pi is ARM architecture which Clickhouse is not compatible with

**Comment by DennisLoska** [comment]
@ukutaht By now they have ARM images so it would be cool if you could create one on your dockerhub as well. https://hub.docker.com/r/clickhouse/clickhouse-server/tags

**Comment by DennisLoska** [comment]
@GrahamSH-LLK It's working with this image on RPI4 https://github.com/plausible/analytics/pull/2103

**Reply by GrahamSH-LLK** [reply]
Nice! At this point I've moved on to actual infrastructure but that's great for running on ARM servers.
